finance/hledger-iadd/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           haskell_stack 1.0
+
+github.setup        hpdeifel hledger-iadd 2afeaadbbb29b339b48799ed2bd3112193d64f25
+version             1.3.21
+revision            2
```

Comment:
A brand new Portfile should have a `revision` of zero.  `revision` is increased mainly if there needs to be a change to a port where it's actual `version` isn't changing.

Reply:
Fixed